Mathlib/Order/Filter/Prod.lean

```diff
@@ -495,9 +509,9 @@ theorem map_const_principal_coprod_map_id_principal {α β ι : Type*} (a : α)
 the identity function.  Together with the previous lemma,
 `map_const_principal_coprod_map_id_principal`, this provides an example showing that the inequality
 in the lemma `map_prod_map_coprod_le` can be strict. -/
```

Comment:
Please also update this doc-string, thanks! (And can you check if there are remaining occurrences of `prod_map` in mathlib?)